docs/google_provider_setup.md

```diff
@@ -122,7 +122,7 @@ have been filled in, click the "Authorize" button.
 To give `sso-auth` permission to access Google Group information for users, the following
 environment variables must be set:
 
-- **`GOOGLE_ADMIN_EMAIL`**: An administrative email address on your organization's
+- **`PROVIDER_GOOGLE_GOOGLE_IMPERSONATE`**: An administrative email address on your organization's
```

Comment:
The first `GOOGLE` here doesn't have to be `GOOGLE` -- it can be any unique identifier that logical groups a set of provider configs together. This is confusing but maybe figuring out a way to call out?

Reply:
Yeah..Let me note that specifically and try to make that a bit clearer - maybe just using a different unique identifier for the quickstart will help.

Reply:
@jphines - update the documentation to hopefully make this clearer